README.md

```diff
@@ -185,6 +185,97 @@ cargo run --example publish_subscribe_publisher
 cargo run --example publish_subscribe_subscriber
 ```
 
+### Request Response
+
+This example showcases a client sending a request with the number 123 every
+second, while a server responds with the number 456 as soon as it receives
+the request.
+
+#### client.rs
+
+```rust
+use core::time::Duration;
+use iceoryx2::prelude::*;
+
+const CYCLE_TIME: Duration = Duration::from_secs(1);
+
+fn main() -> Result<(), Box<dyn core::error::Error>> {
+    let node = NodeBuilder::new().create::<ipc::Service>()?;
+
+    let service = node
+        .service_builder(&"My/Funk/ServiceName".try_into()?)
+        .request_response::<u64, u64>()
+        .open_or_create()?;
+
+    let client = service.client_builder().create()?;
+
+    // send first request
+    let request = client.loan_uninit()?;
+    let request = request.write_payload(1234);
```

Comment:
Not important, but my inner chicken wants the payload `123` here :wink:

Reply:
In pub sub it is also 1234 so we keep it for consistency ;)